lib/ufe/StagesSubject.cpp

```diff
@@ -207,6 +224,20 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 	{
 		auto usdPrimPathStr = changedPath.GetPrimPath().GetString();
 		auto ufePath = stagePath(sender) + Ufe::PathSegment(usdPrimPathStr, g_USDRtid, '/');
+
+        // isPrimPropertyPath() does not consider relational attributes
+        // isPropertyPath() does consider relational attributes
+        // isRelationalAttributePath() considers only relational attributes
+        if (changedPath.IsPrimPropertyPath()) {
+            if (inAttributeChangedNotificationGuard()) {
+                pendingAttributeChangedNotifications[ufePath] =
+                    changedPath.GetName();
+            }
+            else {
+                Ufe::Attributes::notify(ufePath, changedPath.GetName());
+            }
+        }
```

Comment:
See comments in UsdAttribute.cpp on why this is needed.  Essentially, USD sends two USD notifications for a single set attribute call, so we collapse this down to a single UFE notification.  This is done through use of a guard that restricts the scope of notification collapsing to a single USD set attribute, and a map that does the notification collapsing.